### PR TITLE
[12.0] attachment_s3: Add set ACL function

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

**Review by jcoux — APPROVED** [review]
Seems good, but I'm not an expert.

File: attachment_s3/models/ir_attachment.py [hunk 1]
```diff
@@ -106,6 +106,34 @@ def _get_s3_bucket(self, name=None):
...
+            except exceptions.UserError:
+                _logger.exception(
+                    "error getting bucket from object storage"
+                )
+            key = s3uri.item()
+            try:
+                obj = bucket.Object(key=key)
+                obj.Acl().put(ACL=acl)
+                _logger.info(
+                    "ACL public-read successfully set on object %s" % fname
```
Comment (p-tombez) [inline attachment_s3/models/ir_attachment.py]:
```suggestion
                    "ACL %s successfully set on object %s" % (acl, fname)
```

**Review by p-tombez — APPROVED** [review]
LG, just a bit of nitpicking

**Comment by guewen** [comment]
@grindtildeath Can you squash the fixups?

**Comment by grindtildeath** [comment]
@guewen Done :)

**Comment by leemannd** [comment]
This PR aims to have ACL -> we will remove this by schema application in S3 bucket

**Comment by grindtildeath** [comment]
PR closed as we move from ACL per file to bucket policies using prefixes.

**Pull request closed without merging**